ORM/Connect/MySQLSchemaManager.php

```diff
@@ -454,8 +459,8 @@ public function set($values) {
 		//DB::requireField($this->tableName, $this->name, "enum('" . implode("','", $this->enum) . "') character set
 		//utf8 collate utf8_general_ci default '{$this->default}'");
 		$valuesString = implode(",", Convert::raw2sql($values['enums'], true));
-		$charset = Config::inst()->get('MySQLDatabase', 'charset');
-		$collation = Config::inst()->get('MySQLDatabase', 'collation');
+		$charset = Config::inst()->get('SilverStripe\ORM\Connect\MySQLDatabase', 'charset');
+		$collation = Config::inst()->get('SilverStripe\ORM\Connect\MySQLDatabase', 'collation');
 		return "set($valuesString) character set {$charset} collate {$collation}" . $this->defaultClause($values);
```

Comment:
After seeing a lot of fully namespaced items. I think, it would add value to be able to call Config without using full namespace. Do you see an option there?